howtos/generic_ci_artifactory.rst

```diff
@@ -91,7 +91,7 @@ is calling to the ``start`` subcommand of ``conan_build_info``. This will set th
     $ cd .. && cd consumer
     $ conan install . # Consumes mypackage, generates a lockfile
     $ conan create . consumer/1.0@user/stable --lockfile conan.lock
-    $ conan upload * -c -r local # Upload all packages to local remotes
+    $ conan upload '*' -c -r local # Upload all packages to local remotes
```

Comment:
I am afraid this is broken in Windows terminal. We need to use double quotes for compatibility

Reply:
```
(conanpip37) C:\Users\memsharded>conan search "*"
Existing package recipes:

openssl/1.0.2t
poco/1.9.4
zlib/1.2.11

(conanpip37) C:\Users\memsharded>conan search '*'
There are no packages matching the ''*'' pattern

(conanpip37) C:\Users\Diego>```

Reply:
Done. (I think, not very familiar with the pull request UI)